tests/ui/privacy/issue-113860.rs

```diff
@@ -0,0 +1,18 @@
+#![allow(bare_trait_objects)]
+//~^ ERROR module has missing stability attribute
+#![feature(staged_api)]
+
+// See #113860.
+
+mod m {
+    pub trait Trait {}
+
+    impl Trait for Trait {
+        //~^ ERROR the object type `(dyn Trait + 'static)` automatically implements the trait `Trait` [E0371]
```

Comment:
Do you think it's important to keep the original reproduction?
I don't like it due to the "off-topic" errors like this, the minimized test is probably enough.

Reply:
I suppose not - I've removed it.